const getDivisions = async (req, res, next) => {
    try {
        
    } catch (error) {

    }
}